examples/sundials/ex16p.cpp

```diff
@@ -1,21 +1,25 @@
 //                         MFEM Example 16 - Parallel Version
 //                              SUNDIALS Modification
 //
-// Compile with: make ex16p
+// Compile with: make sundials_ex16p
```

Comment:
```suggestion
// Compile with:
//    make ex16p            (GNU make)
//    make sundials_ex16p   (CMake)
```

Reply:
With the update in `CMakeLists.txt` so the executables don't include the `sundials_` prefix to be consistent with GNU build, I believe this suggestion is no longer needed?

Reply:
Isn't the CMake target still called `sundials_ex16p`? If it does not have the `sundials_` prefix, it will probably conflict with the regular Example 16p which uses the target name `ex16p`. I'm not 100% certain, but I think with CMake, the target names are global unlike GNU make where the targets are specific to the directory.

It's best to just try it with CMake: does `make ex16p`, run from the `examples/sundials` directory, build the SUNDIALS example in that directory, or the regular Example 16p in the `examples` directory?

Reply:
Ah, good catch.  I had tried `make ex16p` after the executable was already created.  I'll apply the suggestion.

Reply:
Err, the suggestion couldn't be applied "as-is" because of other commits.  I introduced the changed (and updated the comments in the other SUNDIALS examples) in a separate commit.